app-office/moneydance/files/moneydance-bin

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+# Find JAVA
+JAVA_OPTS="-client -Dawt.useSystemAAFontSettings=gasp -Dawt.useSystemAAFontSettings=on -Xmx2048m"
+MONEYDANCE_LIBS="/usr/share/moneydance/lib"
+OPENJDK11="/usr/lib/jvm/openjdk-11"
+OPENJDK11BIN="$(find /opt -name 'openjdk-bin-11.*' -type d | head -n1)"
```

Comment:
Uuuugly. Try something like:

    set -- /opt/openjdk-bin-11.* /usr/$(get_libdir)/openjdk-11 ...
    while ! [ -d "${1}" ]; do
        if [ ${#} -eq 1 ]; then
             # FAIL
        fi
        shift
    done

Reply:
Updated script. But how do I add ```$(get_libdir)``` to an ordinary script? I thought this is some ebuild function? Is this even neede, since the eBuild is only ```amd64```?

Reply:
Aye, right. You could technically sed it but it's probably simpler to take `/usr/lib*` ;-).

Reply:
Okay, added.